File: 240201/배열의 특정 위치 찾기 4/find-specific-location-of-array-4.py
arr = list(map(int, input().split()))

new_arr = []
for i in arr:
    if i != 0:
        if i % 2 == 0:
            new_arr.append(i)
    else:
        break

sum_arr = sum(new_arr)
l = len(new_arr)

print(l, sum_arr)

File: 240921/배열의 특정 위치 찾기 4/find-specific-location-of-array-4.cpp
#include <iostream>
using namespace std;

int main() {
    // 여기에 코드를 작성해주세요.
    int cnt = 0;
    int sum = 0;
    int arr[10];
    for (int i = 0; i < 10; i ++) {
        cin >> arr[i];
        if (arr[i] == 0) {
            break;
        }
        else {
            if (arr[i] % 2 == 0) {
                cnt ++;
                sum += arr[i];
            }
        }
    }
    cout << cnt << " " << sum;
    return 0;
}